config/karma.conf.js

```diff
@@ -0,0 +1,74 @@
+const fs = require('fs');
+
+// read https key/cert
+const httpsOptions = {};
+try {
+  httpsOptions.cert = fs.readFileSync(`${__dirname}/certs/cert.pem`);
+  httpsOptions.key = fs.readFileSync(`${__dirname}/certs/key.pem`);
+} catch (e) {
+  console.log(e);
+  process.exit(1);
+}
+
+const webpackConfigs = require('./webpack');
+
+module.exports = function(config) {
+
+  const browserFlags = [
+    '--ignore-certificate-errors',
+    '--disable-web-security',
+    '--no-sandbox',
+    '--disable-gpu',
+    '--headless',
+  ];
+
+  config.set({
+    // Project configurations
+    basePath: '..',
+    frameworks: ['mocha', 'chai'],
+    files: [
+      'test/hardwareWallet.js',
+    ],
+    exclude: ['node_modules/**/test/*.js'],
+    preprocessors: {
+      'test/hardwareWallet.js': ['webpack']
```

Comment:
can this and the `files` prop be generalized so it can be dropped in to other projects?

Reply:
Yes, good point, that got left in there from testing

Reply:
👍